converter/convertBIN2csv.m

```diff
@@ -32,6 +37,16 @@
 while read
     packet = parseBtPacket(fid);
     switch packet.type
+        case 'dev_info'
+            if ~exist('device_info', 'var') || isempty(device_info)
+                    disp('No device info discovered')
```

Comment:
I am not sure why this message is printed. In my test, when the packet is a device_info packet, the program goes into this case and then the message `No device info discovered` is shown while the current packet is actually a device info packet!

Reply:
Good point! Bad correction by me. Altered now